sys-power/acpilight/files/acpilight.initd

```diff
@@ -0,0 +1,45 @@
+#!/sbin/openrc-run
+# $Id$
```

Comment:
From the discussion on [bug 611234](https://bugs.gentoo.org/show_bug.cgi?id=611234), it sounds like repoman should have caught this. I'll file a bug.